nifi-nar-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/src/main/java/org/apache/nifi/dbcp/DBCPConnectionPool.java

```diff
@@ -398,6 +405,75 @@ protected PropertyDescriptor getSupportedDynamicPropertyDescriptor(final String
         return results;
     }
 
+    @Override
+    public List<ConfigVerificationResult> verify(final ConfigurationContext context, final ComponentLog verificationLogger, final Map<String, String> variables) {
+        List<ConfigVerificationResult> results = new ArrayList<>();
+
+        KerberosUser kerberosUser = null;
+        try {
+            kerberosUser = getKerberosUser(context, Collections.emptyMap());
+            if (kerberosUser != null) {
+                results.add(new ConfigVerificationResult.Builder()
+                        .verificationStepName("Configure Kerberos User")
+                        .outcome(SUCCESSFUL)
+                        .explanation("Successfully configured Kerberos user")
+                        .build());
+            }
+        } catch (final Exception e) {
+            verificationLogger.error("Failed to configure Kerberos user", e);
+            results.add(new ConfigVerificationResult.Builder()
+                    .verificationStepName("Configure Kerberos User")
+                    .outcome(FAILED)
+                    .explanation("Failed to configure Kerberos user: " + e.getMessage())
+                    .build());
+        }
+
+        final BasicDataSource dataSource = new BasicDataSource();
+        try {
+            configureDataSource(dataSource, kerberosUser, context, variables);
+            results.add(new ConfigVerificationResult.Builder()
```

Comment:
Does this cause the Verify button to return success? I put in a bad URL and a bad Driver name and it still passed verification. However it gave the expected error when I hit Apply (or Enable, can't remember)

Reply:
No, this will just add one successful result, but the other steps should fail.  I'm surprised, because I can easily get it to fail with a bad URL or driver -- can you share the values you used?

Reply:
I must've had a bad build, I did a full rebuild and everything worked as expected, thanks!